overlord/ifacestate/handlers.go

```diff
@@ -543,12 +548,18 @@ func (m *InterfaceManager) doConnect(task *state.Task, _ *tomb.Tomb) (err error)
 		return fmt.Errorf("snap %q has no %q plug", connRef.PlugRef.Snap, connRef.PlugRef.Name)
 	}
 
+	// TODO: should the repo return an app set here?
```

Comment:
yes, when we later do larger changes to it yes, let's make sure that TODO about this contain one consistent string like "app set" and keep note of this